src/emulator/CMakeLists.txt

```
@@ -49,6 +49,10 @@ if (USE_GDBSTUB)
 	add_definitions(-DUSE_GDBSTUB)
 endif()
 
+if (USE_DISCORD_RICH_PRESENCE)
+	add_definitions(-DDISCORD_RICH_PRESENCE)
```

```suggestion
	add_definitions(-DUSE_DISCORD_RICH_PRESENCE)
```
kind of nitpicky but gdbstub is done the same way